### DISCUSSION TITLE
%E7%BD%91%E7%AB%99%E7%9A%84%E8%B4%A1%E7%8C%AE%E8%80%85/Unlinearity/

### DISCUSSION THREAD

**Discussion by giscus** [UNLINEARITY/Learn-Everything#8] [Q&A]
# %E7%BD%91%E7%AB%99%E7%9A%84%E8%B4%A1%E7%8C%AE%E8%80%85/Unlinearity/

https://www.unlinearity.top/%E7%BD%91%E7%AB%99%E7%9A%84%E8%B4%A1%E7%8C%AE%E8%80%85/Unlinearity/

**Comment by YANYANYANYANZI** [comment]
做多少，就要想多少。
不过无所谓，启动！

**Comment by quietwind1** [comment]
牛的
